packages/mdc-textfield/_mixins.scss

```diff
@@ -438,3 +426,29 @@
     @include mdc-theme-prop(color, $color);
   }
 }
+
+@mixin mdc-text-field-box-background-color_($color) {
+  @include mdc-theme-prop(background-color, $color);
+}
+
+@mixin mdc-text-field-textarea-stroke-color_($color) {
+  @include mdc-theme-prop(border-color, $color);
+}
+
+@mixin mdc-text-field-textarea-background-color_($color) {
+  @include mdc-theme-prop(background-color, $color);
+}
+
+@mixin mdc-text-field-focused-textarea-stroke-color_($color) {
+  .mdc-text-field__input {
+    @include mdc-theme-prop(border-color, $color);
+  }
+
+  @include mdc-theme-prop(border-color, $color);
+}
+
+@mixin mdc-text-field-fullwidth-bottom-line-color_($color) {
+  &.mdc-text-field--fullwidth:not(.mdc-text-field--textarea) {
```

Comment:
`.mdc-text-field--fullwidth` is unnecessarily specific here. Assume the developer is already using a CSS selector that only selects `mdc-text-field--fullwidth` elements